.github/workflows/init.yaml

```diff
@@ -0,0 +1,149 @@
+name: Initialize
+
+on: 
+  workflow_call:
+    inputs:
+      release:
+        type: string
+    outputs:
+      docker_build_is_latest:
+        description: "Is used to set the tags latest and stable"
+        value: ${{ jobs.init.outputs.is_latest_tag }}
+      docker_build_is_version:
+        description: "Is used to set the version tags: major, major-minor, major-minor-patch"
+        value: ${{ jobs.init.outputs.is_version_tag }}
+      docker_test_tag:
+        description: "Is used to smoke test the latest push image"
+        value: ${{ jobs.init.outputs.docker_tag }}
+      release_new_version:
+        description: "new_version, is a release information"
+        value: ${{ jobs.init.outputs.new_version }}
+      release_latest_version:
+        description: "latest_version, is a release information"
+        value: ${{ jobs.init.outputs.latest_version }}
+      release_kind:
+        description: "release_kind, is a release information"
+        value: ${{ jobs.init.outputs.release_kind }}
+      release_ref:
+        description: "release_ref, is a release information"
+        value: ${{ jobs.init.outputs.release_ref }}
+      release_project:
+        description: "The name of the project to be released"
+        value: ${{ jobs.init.outputs.project }}
+
+jobs:
+  init:
+    runs-on: ubuntu-latest
+    outputs:
+      is_latest_tag: ${{ steps.version.outputs.is_latest_tag }}
+      is_version_tag: ${{ steps.version.outputs.is_version_tag }}
+      docker_tag: ${{ steps.version.outputs.docker_tag }}
+      new_version: ${{ steps.version.outputs.new_version }}
+      latest_version: ${{ steps.version.outputs.latest_version }}
+      release_kind:  ${{ steps.version.outputs.release_kind }}
+      release_ref:  ${{ steps.version.outputs.release_ref }}
+      project:  ${{ steps.version.outputs.project}}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: "set IS_VERSION_TAG"
```

Comment:
Would be nice to use https://github.com/greenbone/actions/tree/main/is-latest-tag here instead

Reply:
i fail to understand why that would be beneficial.